cmd/main.go

```diff
@@ -178,6 +178,7 @@ func InitLogs() *logrus.Entry {
 	if err != nil {
 		logger.Error("Invalid Log Level: ", Options.LogConfig.LogLevel)
 	} else {
+		logger.Level = logLevel
 		log.SetLevel(logLevel)
```

Comment:
I think it pretty confusing to see two methods for setting log-level for two different objects
So yes, I think we should refactor it a bit